scripts/extract-monorepo-docs.sh

```diff
@@ -3,7 +3,7 @@
 # Pull the monorepo docs from the branch specified as the first argument, or `master`.
 
 MONOREPO_CODELOAD_URL='https://codeload.github.com/storybookjs/storybook/tar.gz'
-BRANCH=${1:-6.0-docs}
+BRANCH=${1:-next}
```

Comment:
This seems right at the moment, but will it eventually be `master`?

Reply:
Good question. I think probably the default should be `next` and we should set it specifically when the deploying branch is master of that makes sense